### DISCUSSION TITLE
Papers that look interesting?

### DISCUSSION THREAD

**Discussion by sr320** [sr320/ceabigr#6] [General]
Please add suggestions.. of particular are papers that integrate multiple omic datasets

**Comment by sr320** [comment]
via @hputnam

Gene-based predictive models of trophic modes suggest Asgard archaea are not phagocytotic
https://www.nature.com/articles/s41559-018-0477-7

**Comment by AHuffmyer** [comment]
MiMeNet: Exploring microbiome-metabolome relationships using neural networks
https://journals.plos.org/ploscompbiol/article?id=10.1371/journal.pcbi.1009021

**Comment by AHuffmyer** [comment]
The simple fool's guide to population genomics via RNA-Seq: an introduction to high-throughput sequencing data analysis https://pubmed.ncbi.nlm.nih.gov/22931062/

**Comment by AHuffmyer** [comment]
SNP Discovery Using Next Generation Transcriptomic Sequencing
https://pubmed.ncbi.nlm.nih.gov/27460371/

**Comment by shellywanamaker** [comment]
Systems Biology and Multi-Omics Integration: Viewpoints from the Metabolomics Research Community

https://www.ncbi.nlm.nih.gov/pmc/articles/PMC6523452/

**Comment by ksil91** [comment]
Methods of integrating data to uncover genotype–phenotype interactions
https://www.nature.com/articles/nrg3868
